### ISSUE TITLE
Pre Trained Model

### ISSUE THREAD

**Issue by brappazzo** [ibab/tensorflow-wavenet#307]
Does anyone have a pre trained model they could share? Specifically one trained on music (piano) data?

**Comment by DiyuanLu** [comment]
This is a model trained with 72000+ steps with original configuration using VCTK corpus. Still far from fully trained, but could save sometime.
only changed SILENCE_TRESHOLD = 0.1 other than original 0.3
[2017-12-04T13-48-11.zip](https://github.com/ibab/tensorflow-wavenet/files/1543096/2017-12-04T13-48-11.zip)

copy the model to the path "logdir/train/" and run it by type
`python3 train.py --data_dir=corpus --gc_channels=32 --restore_from logdir/train/2017-12-04T13-48-11`

**Comment by skywolf829** [comment]
@DiyuanLu what was the loss you were getting for this model? Just curious - I'm around 40k steps and its between 1.7 and 2.5. I think I need to adjust some parameters.

**Comment by dreamibor** [comment]
@DiyuanLu How is your result of the final trained model? Does SILENCE_TRESHOLD = 0.1 work better than the original 0.3?

**Comment by DiyuanLu** [comment]
@skywolf829 , I think the loss I got is similar. later around 60K, I think I got around 1.2.
